tests/test_data/poa_datasets.csv

```diff
@@ -5,5 +5,5 @@
 "21090","14874","10.7554/eLife.14874","LTLTxmlGTGTLTLTdatasetsGTGTLTLTdatasets_indGTGT0LTLT/datasets_indGTGTLTLT/datasetsGTGTLTLTprev_published_datasetsGTGTLTLTdatasets_indGTGT0LTLT/datasets_indGTGTLTLT/prev_published_datasetsGTGTLTLT/xmlGTGT"
-"21092","14997","10.7554/eLife.14997","LTLTxmlGTGTLTLTdata_availability_textboxGTGTData Availability text here &amp; suchLTLT/data_availability_textboxGTGTLTLTdatasetsGTGTLTLTdatasetGTGTLTLTseq_noGTGT1LTLT/seq_noGTGTLTLTauthors_text_listGTGTCembrowski M, Spruston NLTLT/authors_text_listGTGTLTLTidGTGThttp://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=adsveykeprejbej&amp;acc=GSE74985LTLT/idGTGTLTLTlicense_infoGTGTGSE74985LTLT/license_infoGTGTLTLTtitleGTGTHipposeq: an RNA-seq based atlas of gene expression in excitatory hippocampal neuronsLTLT/titleGTGTLTLTyearGTGT2016LTLT/yearGTGTLTLT/datasetGTGTLTLTdatasets_indGTGT1LTLT/datasets_indGTGTLTLTreporting_standards_indGTGT0LTLT/reporting_standards_indGTGTLTLT/datasetsGTGTLTLTprev_published_datasetsGTGTLTLTdatasetGTGTLTLTseq_noGTGT1LTLT/seq_noGTGTLTLTauthors_text_listGTGTCembrowski M, Spruston N,LTLT/authors_text_listGTGTLTLTidGTGThttp://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67403LTLT/idGTGTLTLTlicense_infoGTGTGSE67403LTLT/license_infoGTGTLTLTtitleGTGTSpatial gene expression gradients underlie prominent heterogeneity of CA1 pyramidal neuronsLTLT/titleGTGTLTLTyearGTGT2016LTLT/yearGTGTLTLT/datasetGTGTLTLTdatasets_indGTGT1LTLT/datasets_indGTGTLTLT/prev_published_datasetsGTGTLTLT/xmlGTGT"
+"21092","14997","10.7554/eLife.14997","LTLTxmlGTGTLTLTdata_availability_textboxGTGTData Availability text <i>here</i> &amp; suchLTLT/data_availability_textboxGTGTLTLTdatasetsGTGTLTLTdatasetGTGTLTLTseq_noGTGT1LTLT/seq_noGTGTLTLTauthors_text_listGTGTCembrowski M, Spruston NLTLT/authors_text_listGTGTLTLTidGTGThttp://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?token=adsveykeprejbej&amp;acc=GSE74985LTLT/idGTGTLTLTlicense_infoGTGTGSE74985LTLT/license_infoGTGTLTLTtitleGTGTHipposeq: an RNA-seq based atlas of gene expression in excitatory hippocampal neuronsLTLT/titleGTGTLTLTyearGTGT2016LTLT/yearGTGTLTLT/datasetGTGTLTLTdatasets_indGTGT1LTLT/datasets_indGTGTLTLTreporting_standards_indGTGT0LTLT/reporting_standards_indGTGTLTLT/datasetsGTGTLTLTprev_published_datasetsGTGTLTLTdatasetGTGTLTLTseq_noGTGT1LTLT/seq_noGTGTLTLTauthors_text_listGTGTCembrowski M, Spruston N,LTLT/authors_text_listGTGTLTLTidGTGThttp://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67403LTLT/idGTGTLTLTlicense_infoGTGTGSE67403LTLT/license_infoGTGTLTLTtitleGTGTSpatial gene expression gradients underlie prominent heterogeneity of CA1 pyramidal neuronsLTLT/titleGTGTLTLTyearGTGT2016LTLT/yearGTGTLTLT/datasetGTGTLTLTdatasets_indGTGT1LTLT/datasets_indGTGTLTLT/prev_published_datasetsGTGTLTLT/xmlGTGT"
```

Comment:
unfortunately the readability of these examples is reduced by its format, but it's something that comes out of ejp and hence we have to conform to it I guess

Reply:
(the only alternative in the future would be to generate a CSV fixture from a definition written with Python code, but it may constrain too much what we can use)

Reply:
I don't see too many good ways around the test data as it appears in reality. If we could receive cleaner more structured data we would use it. The test data has been altered to make it as simple as possible and to test multiple logic situations. If we added extra whitespace, for example, we'd have to remove it all in order to run the tests and get character perfect output (just one thought).

Generating the data is interesting, and if you'd like to create an issue we could discuss further. Some additional ideas along those lines, I could see maybe if we created a routine that would prettify the data for viewing, but leaves the base data alone? That, or we have something that would validate or convert the data, in these cases the ``LTLT`` and ``GTGT`` being the escape sequences for angle brackets, so that it is easier to understand and find typos.